Example/AppDelegate.swift

```diff
@@ -58,14 +40,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         UIControl.swizzleSendAction()
     }
 
-    func deepLinkToMiniApp(using token: String) {
-        let rootController = window?.rootViewController as? UINavigationController
-        guard let controllersStack = rootController?.viewControllers else { return }
-        if let homeViewController = controllersStack.first(where: { $0 is ViewController }) as? ViewController {
-            homeViewController.getMiniAppPreviewInfo(previewToken: token, config: Config.current(pinningEnabled: true))
-        } else if let firstLaunchController = controllersStack.first(where: { $0 is FirstLaunchViewController }) as? FirstLaunchViewController {
-            firstLaunchController.previewUsingQRToken = token
-        }
```

Comment:
Hopefully we are handling the deeplink

Reply:
yeah I've moved it here https://github.com/laubengaier/ios-miniapp/blob/feat-5.0.0/remove-old-demo-app-code/Example/SampleApp.swift#L18